src/extension.ts

```diff
@@ -93,7 +93,10 @@ function initCommand(
   context.subscriptions.push(vscode.commands.registerCommand(command, callback));
 }
 
-// DigitalTwin extension part
+function isJsonFile(document: vscode.TextDocument): boolean {
```

Comment:
Looks like this function is only called by initIntelliSense(). So can we put it within initIntelliSense()?